CLI.md

```diff
@@ -605,7 +605,7 @@ Run a GraphQL service that exposes a faucet where users can claim tokens. This g
 * `--limit-rate-until <LIMIT_RATE_UNTIL>` — The end timestamp: The faucet will rate-limit the token supply so it runs out of money no earlier than this
 * `--max-chain-length <MAX_CHAIN_LENGTH>` — The maximum number of blocks in the faucet chain, before a new one is created
 
-  Default value: `100`
+  Default value: `1000`
```

Comment:
Hard to say what the best value could be. Maybe `500` ?

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3863/commits/3c779b48b189df2839fabd7ae3c134153f70d211.